core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -641,6 +641,8 @@ enum class UniqueType(
     ConditionalAfterGeneratingGreatProphet("after generating a Great Prophet", UniqueTarget.Conditional),
 
     ConditionalBuildingBuilt("if [buildingFilter] is constructed", UniqueTarget.Conditional),
+    ConditionalBuildingBuiltAll("if [buildingFilter] is constructed in all [cityFilter] cities", UniqueTarget.Conditional),
+    ConditionalBuildingBuiltAmount("if [buildingFilter] is constructed in [positiveAmount] [cityFilter] cities", UniqueTarget.Conditional),
```

Comment:
"in at least", so people don't think they can make different effects for 1,2,3 etc